include/seqan3/alphabet/composition/union_composition.hpp

```diff
@@ -241,7 +241,7 @@ class union_composition : public alphabet_base<union_composition<alternative_typ
     /*!\name Constructors, destructor and assignment
      * \{
      */
-    constexpr union_composition() = default;
+    constexpr union_composition() noexcept = default;
```

Comment:
If this is defaulted is this also automatically noexcept? (regarding the comment before?)

Reply:
Yes it would be, but it shouldn't be defaulted :scream: It should call `: base_type{} {}` !

Thanks for spotting this.

Reply:
The wrappers did not check for noexcept on the member functions. I have added static assertions (they are more verbose than concept checks and it is something you can easily forget).